### DISCUSSION TITLE
dbeaver Dev in eclipse...

### DISCUSSION THREAD

**Discussion by yangmeng66** [dbeaver/dbeaver#14983] [DBeaver Development]
Hello, everyone. Request assistance

An error has been reported in the test/MANIFEST.MF file.
Bundle 'org.eclipse.swtbot.eclipse.finder' cannot be resolved	MANIFEST.MF	/org.jkiss.dbeaver.ext.test/META-INF	line 12	Plug-in Problem

`Require-Bundle: org.eclipse.ui,
 org.eclipse.core.runtime,
 org.jkiss.dbeaver.core,
 org.jkiss.dbeaver.model,
 org.eclipse.core.resources,
 org.eclipse.swtbot.eclipse.finder,
 org.junit,
 com.sun.jna,
 com.sun.jna.platform`

**Comment by LonwoLonwo** [comment]
As a workaround  - you can just close the test folder in Eclipse.

![2021-12-22 16_28_49-eclipse-workspace2021 - Eclipse IDE](https://user-images.githubusercontent.com/45152336/147100491-b1b2c896-38e4-4ea7-9b2b-9122e67690d2.png)

**Comment by miltonchirinos44** [comment]
The idea is to solve the problem, not to give half solutions, for this reason some answers do not answer our questions.
